engine/executor/dm/api.go

```diff
@@ -50,7 +51,9 @@ func (w *dmWorker) StopWorker(ctx context.Context, msg *dmpkg.StopWorkerMessage)
 	if w.taskID != msg.Task {
 		return errors.Errorf("task id mismatch, get %s, actually %s", msg.Task, w.taskID)
 	}
-	return w.Exit(ctx, w.workerStatus(ctx), nil)
+
+	workerStatus := w.workerStatus(ctx)
```

Comment:
@lance6716 Could you help me check the modification of DM part?
Seems `StopWorker` has no meaning of `Cancel` here.

Reply:
So `e2e_dm_test` always fail me.